wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -56,6 +57,8 @@ import           Pos.Wallet.Web.State (AddressInfo (..), AddressLookupMode (Ever
 import           Pos.Wallet.Web.Util (decodeCTypeOrFail, getAccountAddrsOrThrow,
                                       getWalletAccountIds, getWalletAddrsDetector)
 
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
```

Comment:
If there's duplication, we should probably be trying to remove it. Why are you ignoring this hint here? If it's just for WIP stuff then let's remember to remove it.